snapshot/snapshot.go

```diff
@@ -302,7 +361,15 @@ func (o *snapshotter) Prepare(ctx context.Context, key, parent string, opts ...s
 	}
 
 	target, ok := base.Labels[targetSnapshotLabel]
+	// !ok means we are in an active snapshot
 	if !ok {
+		// Setup id-mapped mounts if config allows.
+		// Any error here needs to stop the container from starting.
+		if o.allowIDMap {
```

Comment:
lets say the allowIDMap config is not true then we should return false in case we find the labels?

Reply:
You're right, good catch

Reply:
Rethinking this, though, is there ever a reason to disable id-mapping? It's a new feature sure but in theory it's entirely opt-in behavior that is only triggered by adding the labels.

Maybe we should get rid of this option in the config entirely.

Reply:
yes it should not be required.